common/src/main/java/tachyon/exception/TachyonExceptionType.java

```diff
@@ -24,12 +24,13 @@
   FAILED_TO_CHECKPOINT(FailedToCheckpointException.class),
   FILE_ALREADY_EXISTS(FileAlreadyExistsException.class),
   FILE_DOES_NOT_EXIST(FileDoesNotExistException.class),
+  FILE_ALREADY_COMPLETES(FileAlreadyCompletedException.class),
```

Comment:
FILE_ALREADY_COMPLETES -> FILE_ALREADY_COMPLETED

Also, sort it alphabetically?